docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  comfy-ui:
+    build:
+      context: ./
+      # AMD Users uncomment:
+      dockerfile: ./AMD.Dockerfile
+      # NVIDIA Users uncomment:
+      #dockerfile: ./NVIDIA.Dockerfile
+    restart: unless-stopped
+    volumes:
+      - ./models/checkpoints:/ComfyUI/models/checkpoints
+      - ./models/vae:/ComfyUI/models/vae
+      - ./models/unet:/ComfyUI/models/unet
+      - ./models/clip:/ComfyUI/models/clip
+      - $OUTPUTDIRECTORY:ComfyUI/output
+    ports:
+      - 8188:8188
+    devices:
+      - "/dev/dri:/dev/dri"
+      ## For AMD GPU
+      - "/dev/kfd:/dev/kfd"
```

Comment:
Are any nvidia devices needed or is it assumed that the nvidia docker binary manages this?

On my machine (1050TI, 560.35.03, CUDA 12.6, Kernel 6.8.0), it looks like this:
```
$ lsof -p $(pidof python) 2>/dev/null | grep /dev | grep -v -e zero -e pts -e shm  | awk '{print $9}' | sort -u 
/dev/nvidia0
/dev/nvidia-caps/nvidia-cap2
/dev/nvidiactl
/dev/nvidia-uvm
```

Reply:
Nvidia GPU's should be made directly aviable under /dev/dri by the linux kernel, as far as i know.